src/utilities/io.cpp

```diff
@@ -435,7 +447,7 @@ void incflo::WritePlotFile()
         pltscaVarsName.push_back("gpx");
         ++icomp;
     }
-    if (m_plt_gpy) {
+if (m_plt_gpy) {
```

Comment:
```suggestion
    if (m_plt_gpy) {
```

Reply:
The spacing on this is funny. Not completely sure that the suggestion fixes it.

Reply:
Done